src/05-WorkingWithData.Rmd

```diff
@@ -1123,6 +1123,204 @@ Any feedback for this section? Click [here](https://docs.google.com/forms/d/e/1F
 
 ## Vignettes
 
+```{r, include = F}
+
+#This may break everything for you alex. Unclear. Helps print help files. 
+library(printr)
+
+```
```

Comment:
This works, but I'd rather not print help inline

Reply:
That sounds good to me. Do you want to merge and pull it out or have me do that?